src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("The truck is working");
```

Comment:
The message 'The truck is working' could be more informative. Consider including the type of machine that is working, for example, 'Truck is working.'